scripts/updateGeneratedFiles.ts

```diff
@@ -142,6 +150,43 @@ async function main({ outDir, rosOutDir }: { outDir: string; rosOutDir: string }
       generateMarkdown(Object.values(foxgloveMessageSchemas), Object.values(foxgloveEnumSchemas)),
     );
   });
+
+  await logProgress("Generating Pyclass definitions", async () => {
+    const dir = await fs.mkdir(path.join(outDir, "pyclass"), { recursive: true });
```

Comment:
I suggest writing these directly to the python directory, like we're doing for the typescript types.

The top level `schemas` directory is really just to make it easy for people to find if they want to copy/paste the .proto or other files into their own project. I can't think of any other reason to save the generated python code here.

We should try to keep the generated code in its own directory though, so we can add a top level `yarn clean`. I'll make a PR with that.

Reply:
Update: https://github.com/foxglove/foxglove-sdk/pull/202

Reply:
This is done now. I wonder if this shouldn't even be part of the same `update` task; they feel spiritually different, and contributors to `schemas` don't need to worry about this one. This is especially true since (per comments below) the pyclass generation now has optional dependencies on cargo, poetry, and black.

Reply:
I cleaned this up with your changes above. For now, I put this behind a `--include-sdk` flag, which keeps the diff smaller. I do think a separate command might be worthwhile, but that could probably wait for FG-10347.